website/docs/components/form/input-group/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,115 @@
+    - [Search](/components/form/text-input#search)
+    - [Date and time](/components/form/text-input#date-and-time)
+- [Select](/components/form/select)
+
+Different Segmented Input types can be combined within an Input Group to collect more complex data and support complex filtering patterns. Use the type that makes the most sense for the intended value or data.
+
+### Custom elements
+
+If you need to use a custom component within an Input Group, ensure that the visual language and styling match the primitives, or contact the Design Systems Team for [assistance](/about/support).
+
+## Base component
+
+In Figma, the `Base` component is a combination of a Segmented Button and Segmented Input and is intended for more simple use cases. The properties of each nested primitive are exposed in the component to support rapid customization.
+
+![Base component example](/assets/components/form/input-group/input-group-base-component.png =500x*)
+
+## Examples
+
+Since an Input Group is intended to be agnostic in relation to the inputs and actions it contains, the number of potential use cases and examples are broad. These examples should serve as inspiration for the different scenarios, contexts, and combinations of elements supported by the Input Group guidelines.
+
+### Within a filter pattern
+
+An Input Group can be used in a filter pattern as a means to reduce or limit the returned items in a dataset through the selection of specific parameters and values that are present in the dataset. Multiple Input Groups may be used within a filter pattern depending on the complexity of the dataset.
+
+![Filter pattern example](/assets/components/form/input-group/input-group-example-filter-pattern-01.png)
+
+This example showcases a larger filter pattern that consists of two Input Groups; the first allowing the user to filter what parameters are included in a search query, the second grouping similar filter parameters relationally.
+
+![Open filter pattern example](/assets/components/form/input-group/input-group-example-filter-pattern-01-open.png)
+
+### Within a form
+
+While less common, an Input Group can be used within a form to collect related data, perform generative functions like creating API keys, and for filtering within a search field.
+
+!!! Do
+
+Use an Input Group to perform a generative function within a form.
+
+![Input Group API key example](/assets/components/form/input-group/input-group-example-form-01.png =450x*)
+!!!
+
+!!! Do
+
+Use an Input Group to determine the intended method of request or connection.
+
+![Input Group method example](/assets/components/form/input-group/input-group-example-form-02.png =450x*)
+!!!
+
+!!! Dont
+
+Don't use an Input Group when fields are related, but aren't stored in the same object upon submission. This example features a payment form with an Input Group and label, but uses a placeholder instead of a label to communicate what information the field collects.
```

Comment:
```suggestion
Don’t use an Input Group when fields are related, but aren’t stored in the same object upon submission. This example features a payment form with an Input Group and label, but uses a placeholder instead of a label to communicate what information the field collects.
```